pkg/utils/utils.go

```diff
@@ -64,6 +64,13 @@ func TrimTrailingNewline(str string) string {
 	return str
 }
 
+// NormalizeLinefeeds - Removes all Windows and Mac style line feeds
+func NormalizeLinefeeds(str string) string {
+	str = strings.Replace(str, "\r\n", "", -1)
```

Comment:
I believe this should be \r\n mapping to "\n" rather than to "". In the case of git diff it seems to make no difference, but if you're `cat`ing a file it will break:

![image](https://user-images.githubusercontent.com/8456633/44341407-26bc6f80-a4cb-11e8-9114-004ca7b59288.png)